opengl_ros_lib/src/depth_image_projector.cpp

```diff
@@ -196,6 +201,12 @@ void DepthImageProjector::Impl::project(cv::Mat& dest, const cv::Mat& color, con
     {
         //Shaders setup
         program_.use();
+        std::array<float, 4> normalized_camera_pos = {
+          depthToMap_[12] / gridMapResolution_ / gridMapWidth_ * 2,
+          depthToMap_[13] / gridMapResolution_ / gridMapHeight_ * 2,
+          depthToMap_[14] / gridMapLayerHeight_ / 2,
```

Comment:
This line should be `depthToMap_[14] / gridMapLayerHeight_ * 2` since -height/2 to height/2 is mapped to -1 to 1 in NDC space.